README.md

```diff
@@ -100,6 +100,7 @@ wsa://com.android.settings
 | Deus Ex GO | 2.1.111374 | ⚠️ | Terrible in-game experience, includes poor performance and low FPS ||
 | DevCheck | 3.39 | ❌ | Blank screen on startup ||
 | Device Info HW | 5.4.1 | ✔️ |||
+| Decibel X | 6.4.2 |	⚠️ |	App crashes |
```

Comment:
```suggestion
| Decibel X | 6.4.2 |	⚠️ | App crashes |
```

Reply:
I apologise for unnecessary indentation!